### PR TITLE
Windows buildbot + collection of bugfixes

### PR DESCRIPTION
**Pull request by fabianp**
The appveyor scripts are adapted from scikit-learn. Main changes here are:

  - Change version number to scikit-learn standards.
  - replace DBL_MAX in primal_cd_fast.pyx with numpy constant.
  - Include tests in distribution (adding ```__init__.py``` for the test directory).
  - name package skcontrib-lightning in setup.py.
  - Add appveyor badge to README.

### CODE REVIEW

**Comment by fabianp** [comment]
See the successful build on my fork: https://github.com/fabianp/lightning
ping @mblondel

File: setup.py [hunk 1]
```diff
@@ -4,20 +4,20 @@
 
 import sys
 import os
+import setuptools
+from numpy.distutils.core import setup
+
 
-DISTNAME = 'lightning'
+DISTNAME = 'skcontrib-lightning'
```
Comment (mblondel) [inline setup.py]: Maybe `sklearn-contrib-lightning`?

Althought it's a bit longer...

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+cache:
+  # Use the appveyor cache to avoid re-downloading large archives such
+  # the MKL numpy and scipy wheels mirrored on a rackspace cloud
+  # container, speed up the appveyor jobs and reduce bandwidth
+  # usage on our rackspace account.
+  - '%APPDATA%\pip\Cache'
+
+deploy:
+  release: sklearn-v0.1-pre
```
Comment (mblondel) [inline appveyor.yml]: sklearn-comtrib-lightning?

**Comment by mblondel** [comment]
+1 for merge apart from my comments

**Comment by fabianp** [comment]
Thanks for the review Mathiew. I've renamed it to sklearn-contrib-lightning. I'll merge it tomorrow if there are no new comments.

**Pull request merged**